diff --git a/node_modules/solid-color/package.json b/node_modules/solid-color/package.json
index cc648ea..fd424d4 100644
--- a/node_modules/solid-color/package.json
+++ b/node_modules/solid-color/package.json
@@ -29,22 +29,8 @@
   "sideEffects": false,
   "type": "module",
   "exports": {
-    "solid": {
-      "development": "./dist/source/index.jsx",
-      "import": "./dist/source/index.jsx"
-    },
-    "development": {
-      "import": {
-        "types": "./dist/types/index.d.ts",
-        "default": "./dist/esm/index.js"
-      },
-      "require": "./dist/cjs/index.js"
-    },
-    "import": {
-      "types": "./dist/types/index.d.ts",
-      "default": "./dist/esm/index.js"
-    },
-    "require": "./dist/cjs/index.js"
+    "development": "./src/index.tsx",
+    "import": "./src/index.tsx"
   },
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
diff --git a/node_modules/solid-color/src/components/_common/Hue.tsx b/node_modules/solid-color/src/components/_common/Hue.tsx
index 54a7bd9..0041fa3 100644
--- a/node_modules/solid-color/src/components/_common/Hue.tsx
+++ b/node_modules/solid-color/src/components/_common/Hue.tsx
@@ -4,6 +4,7 @@ import { HslColor } from '../../types'
 
 interface HueProps {
   children?: JSX.Element
+  class?: string
   direction?: string
   radius?: number | string
   shadow?: string
@@ -26,21 +27,21 @@ export function Hue(_props: HueProps) {
   const styles = () => {
     return {
       hue: {
-        position: 'absolute',
+        position: 'relative',
         inset: '0px',
         'border-radius': typeof props.radius === 'string' ? props.radius : `${props.radius}px`,
-        'box-shadow': props.shadow,
+        // 'box-shadow': props.shadow,
       },
       container: {
         padding: '0 2px',
         position: 'relative',
         height: '100%',
-        'border-radius': typeof props.radius === 'string' ? props.radius : `${props.radius}px`,
+        'border-radius': 'inherit',
       },
       pointer: {
         position: 'absolute',
         left: props.direction === 'vertical' ? '0px' : `${(props.hsl.h * 100) / 360}%`,
-        top: props.direction === 'vertical' ? `${-((props.hsl.h * 100) / 360) + 100}%` : undefined,
+        top: '50%',
       },
       slider: {
         'margin-top': '1px',
@@ -77,7 +78,7 @@ export function Hue(_props: HueProps) {
   onCleanup(() => unbindEventListeners())
 
   return (
-    <div style={styles().hue}>
+    <div style={styles().hue} class={props.class}>
       <div
         ref={container!}
         class={`hue-${props.direction}`}
diff --git a/node_modules/solid-color/src/components/_common/Saturation.tsx b/node_modules/solid-color/src/components/_common/Saturation.tsx
index 8d86b0a..e9b99cb 100644
--- a/node_modules/solid-color/src/components/_common/Saturation.tsx
+++ b/node_modules/solid-color/src/components/_common/Saturation.tsx
@@ -7,6 +7,7 @@ export type SaturationProps = {
   hsl: HslColor
   hsv: HsvColor
   pointer?: JSX.Element
+  class?: string
   onChange?: (color: ChangeColor, event?: Event) => void
   shadow?: JSX.CSSProperties['box-shadow']
   radius?: JSX.CSSProperties['border-radius']
@@ -33,7 +34,7 @@ export function Saturation(_props: SaturationProps) {
     handleChange(event)
 
     if (container) {
-      container.addEventListener('mousemove', handleChange)
+      window.addEventListener('mousemove', handleChange)
       container.addEventListener('mouseup', handleMouseUp)
     }
   }
@@ -44,7 +45,7 @@ export function Saturation(_props: SaturationProps) {
 
   function unbindEventListeners() {
     if (container) {
-      container.removeEventListener('mousemove', handleChange)
+      window.removeEventListener('mousemove', handleChange)
       container.removeEventListener('mouseup', handleMouseUp)
     }
   }
@@ -54,21 +55,21 @@ export function Saturation(_props: SaturationProps) {
     return merge<Record<string, JSX.CSSProperties>, Record<string, JSX.CSSProperties>>(
       {
         color: {
-          position: 'absolute',
+          position: 'relative',
           inset: '0px',
           background: `hsl(${hsl.h},100%, 50%)`,
-          'border-radius': radius,
+          // 'border-radius': radius,
         },
         white: {
           position: 'absolute',
           inset: '0px',
-          'border-radius': radius,
+          'border-radius': 'inherit',
         },
         black: {
           position: 'absolute',
           inset: '0px',
           boxShadow: shadow,
-          'border-radius': radius,
+          'border-radius': 'inherit',
         },
         pointer: {
           position: 'absolute',
@@ -92,6 +93,7 @@ export function Saturation(_props: SaturationProps) {
 
   return (
     <div
+      class={props.class}
       style={styles().color}
       ref={container!}
       onMouseDown={handleMouseDown}
diff --git a/node_modules/solid-color/src/index.tsx b/node_modules/solid-color/src/index.tsx
index 42926e9..97334e9 100644
--- a/node_modules/solid-color/src/index.tsx
+++ b/node_modules/solid-color/src/index.tsx
@@ -13,4 +13,6 @@ export * from './components/chrome'
 export * from './components/sketch'
 export * from './components/photoshop'
 export * from './components/hue'
+export * from './components/_common/Saturation'
+export * from './components/_common/Hue'
 export * from './types'
