diff --git a/node_modules/typescript/lib/_tsc.js b/node_modules/typescript/lib/_tsc.js
index 11ab5ff..7306a42 100644
--- a/node_modules/typescript/lib/_tsc.js
+++ b/node_modules/typescript/lib/_tsc.js
@@ -36430,6 +36430,7 @@ var libEntries = [
   ["esnext.float16", "lib.esnext.float16.d.ts"],
   ["esnext.error", "lib.esnext.error.d.ts"],
   ["esnext.sharedmemory", "lib.esnext.sharedmemory.d.ts"],
+  ["esnext.typedarrays", "lib.esnext.typedarrays.d.ts"],
   ["decorators", "lib.decorators.d.ts"],
   ["decorators.legacy", "lib.decorators.legacy.d.ts"]
 ];
diff --git a/node_modules/typescript/lib/lib.esnext.d.ts b/node_modules/typescript/lib/lib.esnext.d.ts
index c87beab..5aef454 100644
--- a/node_modules/typescript/lib/lib.esnext.d.ts
+++ b/node_modules/typescript/lib/lib.esnext.d.ts
@@ -27,3 +27,4 @@ and limitations under the License.
 /// <reference lib="esnext.float16" />
 /// <reference lib="esnext.error" />
 /// <reference lib="esnext.sharedmemory" />
+/// <reference lib="esnext.typedarrays" />
diff --git a/node_modules/typescript/lib/lib.esnext.typedarrays.d.ts b/node_modules/typescript/lib/lib.esnext.typedarrays.d.ts
new file mode 100644
index 0000000..3c72422
--- /dev/null
+++ b/node_modules/typescript/lib/lib.esnext.typedarrays.d.ts
@@ -0,0 +1,74 @@
+interface Uint8Array<TArrayBuffer extends ArrayBufferLike> {
+    /**
+     * Converts the `Uint8Array` to a base64-encoded string.
+     * @param options If provided, sets the alphabet and padding behavior used.
+     * @returns A base64-encoded string.
+     */
+    toBase64(
+        options?: {
+            alphabet?: "base64" | "base64url" | undefined;
+            omitPadding?: boolean | undefined;
+        },
+    ): string;
+
+    /**
+     * Sets the `Uint8Array` from a base64-encoded string.
+     * @param string The base64-encoded string.
+     * @param options If provided, specifies the alphabet and handling of the last chunk.
+     * @returns An object containing the number of bytes read and written.
+     * @throws {SyntaxError} If the input string contains characters outside the specified alphabet, or if the last
+     * chunk is inconsistent with the `lastChunkHandling` option.
+     */
+    setFromBase64(
+        string: string,
+        options?: {
+            alphabet?: "base64" | "base64url" | undefined;
+            lastChunkHandling?: "loose" | "strict" | "stop-before-partial" | undefined;
+        },
+    ): {
+        read: number;
+        written: number;
+    };
+
+    /**
+     * Converts the `Uint8Array` to a base16-encoded string.
+     * @returns A base16-encoded string.
+     */
+    toHex(): string;
+
+    /**
+     * Sets the `Uint8Array` from a base16-encoded string.
+     * @param string The base16-encoded string.
+     * @returns An object containing the number of bytes read and written.
+     */
+    setFromHex(string: string): {
+        read: number;
+        written: number;
+    };
+}
+
+interface Uint8ArrayConstructor {
+    /**
+     * Creates a new `Uint8Array` from a base64-encoded string.
+     * @param string The base64-encoded string.
+     * @param options If provided, specifies the alphabet and handling of the last chunk.
+     * @returns A new `Uint8Array` instance.
+     * @throws {SyntaxError} If the input string contains characters outside the specified alphabet, or if the last
+     * chunk is inconsistent with the `lastChunkHandling` option.
+     */
+    fromBase64(
+        string: string,
+        options?: {
+            alphabet?: "base64" | "base64url" | undefined;
+            lastChunkHandling?: "loose" | "strict" | "stop-before-partial" | undefined;
+        },
+    ): Uint8Array<ArrayBuffer>;
+
+    /**
+     * Creates a new `Uint8Array` from a base16-encoded string.
+     * @returns A new `Uint8Array` instance.
+     */
+    fromHex(
+        string: string,
+    ): Uint8Array<ArrayBuffer>;
+}
diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index 2643aa1..6b42584 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -40690,7 +40690,8 @@ var libEntries = [
   ["esnext.promise", "lib.esnext.promise.d.ts"],
   ["esnext.float16", "lib.esnext.float16.d.ts"],
   ["esnext.error", "lib.esnext.error.d.ts"],
-  ["esnext.sharedmemory", "lib.esnext.sharedmemory.d.ts"],
+  ["esnext.sharedmemorys", "lib.esnext.sharedmemorys.d.ts"],
+  ["esnext.typedarray", "lib.esnext.typedarray.d.ts"],
   ["decorators", "lib.decorators.d.ts"],
   ["decorators.legacy", "lib.decorators.legacy.d.ts"]
 ];
